src/app/base/components/record-card/record-card-helper.ts

```diff
@@ -0,0 +1,29 @@
+import { DataBrowserModals, RecordCommentsModal } from "src/app/data/components/data-browser/helper-classes/modals-helper";
+import { SimilarSearch } from "src/app/data/components/data-browser/helper-classes/search-similar";
+import { ActiveSearchParams, AttributeSort, DataBrowserRecord } from "./record-card.types";
+
+export type RecordCardOptions = {
+    /**
+     * Options set for kern record card
+     * @attributesSortOrder {AttributeSort[] | any[], optional} - Array of sorted attributes to be displayed
+     * @dataBrowserModals {DataBrowserModals, optional} - Data browser modals with specific fields
+     * @indexExtendedRecord {number, optional} - Index of the extended record in the record list
```

Comment:
i think just calling it recordIndex or currentRecordIndex would be more intuitive since the extendedRecord stuff is only loosely linked to the card component (since its located outside the folder scope)